README.md

```diff
@@ -87,35 +87,137 @@ Usage as a library
 
 The Synapse client can be used to write software that interacts with the Sage Bionetworks Synapse repository.
 
-### Example
+### Examples
 
-    import synapseclient
+#### Log-in and create a Synapse object
+```
+import synapseclient
 
-    syn = synapseclient.Synapse()
+syn = synapseclient.Synapse()
+
+## log in using auth token
+syn.login(authToken='auth_token')
+```
+
+#### Create a manifest TSV and sync a directory to synapse
```

Comment:
NIt: not everyone will know what `manifest` means, so we could potentially simplify this with `sync a local directory to Synapse`.